ballerina-tests/tests/service_native_endpoint_test.bal

```diff
@@ -32,7 +32,8 @@ service /serviceEndpointHello on serviceEndpointTestEP {
 
     resource function get local(http:Caller caller, http:Request req) {
         http:Response res = new;
-        json connectionJson = {local:{host:caller.localAddress.host, port:caller.localAddress.port}};
+        http:Local localAddress = caller.localAddress;
+        json connectionJson = {local:{host:localAddress.host, port:localAddress.port}};
```

Comment:
Is there a special reason for this change ?

Reply:
Reverted with [aa491cc8cd80a6d41f1a04e1dac0c5382dbb25e2](https://github.com/ballerina-platform/module-ballerina-http/pull/765/commits/aa491cc8cd80a6d41f1a04e1dac0c5382dbb25e2)